content/committees/specification/operations/index.md

```diff
@@ -355,6 +358,21 @@ The specification committee will promote the specification project to jakarta.ee
 Links: <https://github.com/jakartaee/jakarta.ee>,
 <https://gohugo.io/documentation/>
 
+### Creating a Final Service Release Specification # {#creating_a_final_service_release_specification}
+    
+There is no formal release review required for a Specification service release (x.y.z) as long as the JESP definition of a "service release" is adhered to.
+That is, no functionality changes are permitted in a service release.
```

Comment:
That is, no functionality changes or increase of IP scope are permitted in a service release.

Reply:
Done.  Thanks!